src/components/layer-controller/LayerOptions.js

```diff
@@ -58,20 +87,51 @@ function LayerOption({ name, inputId, children }) {
 }
 
 function LayerOptions({
-  colormap, opacity, handleColormapChange, handleOpacityChange,
+  colormap,
+  opacity,
+  handleColormapChange,
+  handleOpacityChange,
+  globalControlFields,
+  handleGlobalChannelsSelectionChange,
+  channels,
+  dimensions,
 }) {
   return (
     <Grid container direction="column" style={{ width: '100%' }}>
       <Grid item>
         <LayerOption name="Colormap" inputId="colormap-select">
-          <ColormapSelect value={colormap} inputId="colormap-select" handleChange={handleColormapChange} />
+          <ColormapSelect
+            value={colormap}
+            inputId="colormap-select"
+            handleChange={handleColormapChange}
+          />
         </LayerOption>
       </Grid>
       <Grid item>
         <LayerOption name="Opacity" inputId="opacity-slider">
           <OpacitySlider value={opacity} handleChange={handleOpacityChange} />
         </LayerOption>
       </Grid>
+      {dimensions.length > 0
+        && Object.keys(channels).length > 0
+        && globalControlFields.map(field => (
+          <LayerOption
+            name={field}
+            inputId={`${field}-slider`}
+            key={field}
+          >
+            <GlobalSelectionSlider
+              field={field}
+              value={channels[Object.keys(channels)[0]].selection[field]}
```

Comment:
what does this evaluate to? Why hardcode the first channel?

Reply:
Since it's a global selection, any of the channels will contain the correct value. so I just went for the first one.